go/cmd/vtcombo/main.go

```diff
@@ -78,6 +80,7 @@ func init() {
 	servenv.RegisterGRPCServerFlags()
 	servenv.RegisterGRPCServerAuthFlags()
 	servenv.RegisterServiceMapFlag()
+	servenv.OnParseFor("vtcombo", vschemaacl.RegisterSchemaACLFlags)
```

Comment:
why are these not in `package vschemaacl`? (same comment applies to the identical line in `go/cmd/vtgate/`

Reply:
So I made mistake. I have reverted back my change. Vschemaacl has Init() method which I mixed up with init(). I put registration in Init() and expected the flags to be available for binaries but it was not happening thats why I moved them under each binary endpoint. But now I realized it was because of Init() is not "init()" :D